src/qtbase/QColor.js

```diff
@@ -1,32 +1,78 @@
-// TODO complete implementation (with attributes `r`,`g` and `b`).
-
 class QColor {
   constructor(val) {
+    this.r = this.g = this.b = 0;
```

Comment:
Also notice how the internal `color` representation has more r/g/b/a steps than 256 — those look like floats, which could be important for hsla/hsva representation and `Qt.ligther`/`Qt.darker` (but those are not implemented yet).